src/plugins/explore/public/application/utils/state_management/slices/ui/ui_slice.ts

```diff
@@ -13,7 +13,7 @@ export interface UIState {
 }
 
 const initialState: UIState = {
-  activeTabId: 'logs',
+  activeTabId: '',
   showDatasetFields: true,
   prompt: '', // Initialize prompt as empty string
   showHistogram: true,
```

Comment:
can we change the name of `setActiveTab` to `setActiveTabId` below? :(

Reply:
sure. can change.